publisher/providers/pyth_replicator.py

```diff
@@ -86,3 +122,24 @@ def latest_price(self, symbol: Symbol) -> Optional[Price]:
             return None
 
         return Price(price, conf)
+
+
+def manual_aggregate(prices: List[float]) -> Tuple[float, float]:
+    """
+    This function is used to manually aggregate the prices of the active publishers. This is a very simple
+    implementation that does not get the median and confidence accurately but it is good enough for our use case.
+    """
+    prices.sort()
+    no_prices = len(prices)
+
+    # Ensure the price never goes below zero
+    agg_price = max(0, prices[no_prices // 2])
```

Comment:
i think this check is unnecessary

Also, in my experience, this is the kind of check that will cause you grief later. eg. we add a product that can have a negative price or whatever, and then we see the price is zero, and we forget this check exists, and then it's hard to debug. If there are a set of validity conditions for a certain type of value, you should encode that same set of conditions everywhere. The on-chain program doesn't restrict the price to be nonnegative, so we shouldn't impose that restriction anywhere else either.

(there's some nuance to that rule of thumb, e.g., pyth-agent could add this restriction on a per-product basis if we decide like "hey this is a spot product so it can't be below 0". But the key thing there would be that we know it's a spot product, vs. here it's an arbitrary product.)